### PR TITLE
[10.0] [IMP] base_transaction_id: Add transaction_id on account.payment

### PR DESCRIPTION
**Pull request by lmignon**
This change is required to ease the reconciliation of the account.move
created from the account.payment.
- [ ] required by https://github.com/OCA/sale-workflow/pull/526

### CODE REVIEW

**Review by dvdhinesh — APPROVED** [review]
Code review :+1:

File: base_transaction_id/models/account_payment.py
Comment (adrienpeiffer) [inline base_transaction_id/models/account_payment.py:27]: wouldn't it be better to fill transaction_ref (as is done in this module) ?

**Review by atchuthan — APPROVED** [review]
Code review :+1:

**Comment by Cedric-Pigeon** [comment]
Travis is red because of another addon

**Comment by Cedric-Pigeon** [comment]
@OCA/accounting-maintainers Does anyone can merge this one?

**Review by rousseldenis — APPROVED** [review]
Code review

File: base_transaction_id/models/account_payment.py
Comment (rousseldenis) [reply base_transaction_id/models/account_payment.py:27]: transaction_id is actually the transaction reference ...

**Comment by rousseldenis** [comment]
@sbidoul This one is ready and in production for years

**Comment by sbidoul** [comment]
> @sbidoul This one is ready and in production for years

@rousseldenis @lmignon is it correct that we'll abandon this approach in 13 (and maybe even 12)? In which case it might be better to not merge it?

**Comment by rousseldenis** [comment]
> > @sbidoul This one is ready and in production for years
> 
> @rousseldenis @lmignon is it correct that we'll abandon this approach in 13 (and maybe even 12)? In which case it might be better to not merge it?

Yes, but the module exists already in 10.0 branch. This is an enhancement/fix for the actual installs.

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**
